types/src/dkg/real_dkg/mod.rs

```diff
@@ -39,6 +42,27 @@ pub struct DKGPvssConfig {
     pub pp: DkgPP,
     // DKG encryption public keys
     pub eks: Vec<EncPK>,
+    // Some metrics for caller to consume.
+    #[serde(skip)]
+    pub rounding_summary: RoundingSummary,
```

Comment:
`DKGPvssConfig` is not on-chain right? Do validators ever exchange `DKGPvssConfig`?

Reply:
no i think, i'm not sure why it has the serde derivation. i will see if i can remove them.